src/main/java/io/pakland/mdas/githubstats/domain/service/GetOrganizationFromId.java

```diff
@@ -0,0 +1,27 @@
+package io.pakland.mdas.githubstats.domain.service;
+
+import io.pakland.mdas.githubstats.domain.model.Organization;
+import io.pakland.mdas.githubstats.domain.ports.OrganizationRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+@Service
+public class GetOrganizationFromId {
+
+    final private OrganizationRepository organizationRepository;
+
+    public GetOrganizationFromId(OrganizationRepository organizationRepository) {
+        this.organizationRepository = organizationRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public void execute(Long id) {
+        Optional<Organization> org = organizationRepository.findById(id);
+        if (org.isPresent())
+            System.out.println(org.get());
+        else
+            System.out.println("Organization not found");
```

Comment:
Should we create an exception for this case?

Reply:
Yes, this was just a quick check. It should launch a custom exception, that identify exactly the class where it happened.

Reply:
We'll keep as I have in the tests but we should replace it for an exception.